### ISSUE TITLE
aggregated routes

### ISSUE THREAD

**Issue by DevonGIS** [MIERUNE/GTFS-GO#93]
GTFS GO static data plugin no longer calculates geometry and attributes and geometry from text files created by traffiklab in sweden.
With the API key, a download of all SL data comes as zipped SL.zip data and is then read by the plugin, since some time in april it no longer works.  In addition there are descrepencies when it did calculate aggregates route frequencies in that not as many busses coming to stops and leaving stops are the same amount (and without any good reason, it's on the same bus route).  I would be happy to provide examples if you'd like to contact me

**Comment by takohei** [comment]
Thank you for your report.
Please upload your GTFS data, results, and error message.

**Comment by DevonGIS** [comment]
Ok great, it may be easier if we connect directly, since there are two separate issues and I have graphics and explanations that go with. Is that a possibility?

**Comment by DevonGIS** [comment]
a short description
with these settings:
![image](https://github.com/user-attachments/assets/371988f8-3591-46a0-84a3-25ccfa4de8a7)
we only (since about april) get these two .json layers as vector data
![image](https://github.com/user-attachments/assets/a88bb16a-c517-4350-bdea-5e0029334609)
when we should get vector data for aggregated stops and aggregated routes.  This is gone.
AND.  when we count the number of trips leaving from specific stops (previous or next stop id) the numbers don't match in aggregated routes (from my export last april).
here are busses leaving from a stop, look at frequency
![image](https://github.com/user-attachments/assets/77a68683-c488-49e4-b502-245407a3763f)
here are busses coming to a stop, look at frequency
![image](https://github.com/user-attachments/assets/857213f1-7358-4429-b76a-042914d7325e)

Sometimes the aggregated routes are right, sometimes not, sometimes it's a little bit wrong, sometimes very wrong.  
Gtfs go didn't work right before and now it doesn't work at all.
Please take a look if you plan to continue with this plugin.  Iv'e been testing and looking for months and there's nothing I can change to make it work properly.

Thanks!
devon

**Comment by takohei** [comment]
@DevonGIS 
Thank you for your detailed explanation.
Could you please post the GTFS data or URL so we can reproduce the error?

**Comment by DevonGIS** [comment]
Hi, great, be glad to.  So do you want the raw data, the zip file we get when we download via the api to Traffiklab? or do you want all of the points and lines created by the plugin from the raw data?  do you want the zip file we got in april (that worked to create geometry) as well as the now one that does not? It would be easier if I communicate with you directly, I can make arrangements for a teams meeting or so, regardless of time change.  If this plugin worked it would solve a lot of problems for us

**Comment by takohei** [comment]
>  So do you want the raw data, the zip file we get when we download via the api to Traffiklab?

Yes, please upload the zip file used for  https://github.com/MIERUNE/GTFS-GO/issues/93#issuecomment-2345440398 .

**Comment by DevonGIS** [comment]
I can't load the file because it's 52mb, the limit is 25
